gmfrasca/data-science-pipelines-operator
# V2-argo into DSPv2 Feature Branch

## The issue resolved by this Pull Request:
Resolves #444

## Description of your changes:
Support for v2-argo + v1-tekton

## Testing instructions
- Generate and deploy DSPO from code (`make generate; make deploy IMG=$IMG_FROM_PR`)
- Deploy a v1 DSPA, verify
- `make argodeploy`
- Deploy a v2 DSPA, verify


## Checklist
